eng/test-steps.yml

```diff
@@ -7,7 +7,12 @@ steps:
     displayName: 'Install Node'
     inputs:
       versionSpec: ${{ parameters.version }}
-  - script: npm install
+  - script: npm ci
+    displayName: 'npm ci'
+  - script: npm ls -a
+    displayName: 'npm ls -a'
+    condition: succeededOrFailed()
+    continueOnError: true
```

Comment:
nit: intentional to kep a continueonError here?

Reply:
Yes, we want to list the packages installed, even if `npm ci` fails.